sparse_strips/vello_cpu/src/render.rs

```diff
@@ -81,7 +81,7 @@ pub struct RenderSettings {
 impl Default for RenderSettings {
     fn default() -> Self {
         Self {
-            level: Level::new(),
+            level: Level::try_detect().unwrap_or(Level::fallback()),
```

Comment:
Longer term, we should probably have different handling of this, but for now this is reasonable.